paparazzi/paparazzi/build.gradle

```diff
@@ -26,8 +26,11 @@ dependencies {
 dependencies {
   // Paparazzi is a Kotlin JVM module and thus cannot depend on Android library artifacts (AARs)
   // TODO: Use Gradle transforms to extract/rename classes.jar's from AARs
-  api files('libs/compose-runtime-1.3.1.jar')
-  compileOnly files('libs/compose-ui-1.3.1.jar')
+  api files('libs/compose-runtime-1.3.3.jar')
+  api files('libs/compose-ui-1.3.3.jar')
```

Comment:
```suggestion
  compileOnly files('libs/compose-ui-1.3.3.jar')
```

is there a particular reason why this needs to move from compileOnly to api?

Reply:
nope, reverting

Reply:
So it looks like we do need it to be `api` since we reference `AbstractComposeView` class in the AccessibilityRenderExtension here: https://github.com/cashapp/paparazzi/blob/ff761cf73ca1f17b5a6a4b782d549189cd981bb1/paparazzi/paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt#L80-L80.

Without it, it results to a class not found exception. 
<img width="500" alt="Screenshot 2023-04-25 at 12 00 33 PM" src="https://user-images.githubusercontent.com/18534850/234336273-39e69564-fff6-4b7b-8e18-3f82bd12e694.png">